### PR TITLE
Enable .NET Core 2.0 and .NET Framework 4.6.1 in Microsoft.AI.MachineLearning NuGet package

### PR DESCRIPTION
**Pull request by smk2007**
Enable .NET Core 2.0 and .NET Framework 4.6.1 in Microsoft.AI.MachineLearning NuGet package

Issue: .NET Core 2.0 and .NET Framework 4.6.1 app are not enabled by the Microsoft.AI.MachineLearning NuGet package. This is because
1) There is no projection into C# that wraps that WinRT APIs.
2) The WinRT C# APIs require a projection of the the WinRT library.

Fix: Use the CSWinRT project to generate a c# projection of the Microsoft.AI.MachineLearning API. Add dependency on Microsoft.Windows.SDK.NET for the rest of the WinRT API.

### CODE REVIEW

File: tools/nuget/generate_nuspec_for_native_nuget.py [hunk 1]
```diff
@@ -71,7 +71,24 @@ def generate_repo_url(list, repo_url, commit_id):
 
 
 def generate_dependencies(list, package_name, version):
-    if (package_name != 'Microsoft.AI.MachineLearning'):
+    if (package_name == 'Microsoft.AI.MachineLearning'):
+        list.append('<dependencies>')
+
+        # Support .Net Core
+        list.append('<group targetFramework="NETCOREAPP">')
+        list.append('<dependency id="Microsoft.Windows.SDK.NET"' + ' version="10.0.18362.3-preview"/>')
```
Comment (ryanlai2) [inline tools/nuget/generate_nuspec_for_native_nuget.py:79]: >18362 [](start = 83, length = 5)

How often would this need to get updated?

File: tools/nuget/generate_nuspec_for_native_nuget.py [hunk 2]
```diff
@@ -254,18 +275,21 @@ def generate_files(list, args):
 
     # Process props and targets files
     if is_windowsai_package:
-        # Process props file
-        windowsai_props = os.path.join(args.sources_path, 'csharp', 'src', 'Microsoft.ML.OnnxRuntime',
```
Comment (ryanlai2) [inline tools/nuget/generate_nuspec_for_native_nuget.py:258]: >OnnxRuntime [](start = 89, length = 11)

Why was it Microsoft.ML.Onnxruntime before?

File: tools/nuget/generate_nuspec_for_native_nuget.py (hunk 1 above)
Comment (smk2007) [reply tools/nuget/generate_nuspec_for_native_nuget.py:79]: It is updated when either: 
1) we make deliberate changes to the api and require new SDK functionality. The current and only one available is 18362 - aka 19H1.
2) the preview NuGet is promoted to an official release.

File: tools/nuget/generate_nuspec_for_native_nuget.py (hunk 2 above)
Comment (smk2007) [reply tools/nuget/generate_nuspec_for_native_nuget.py:258]: There was only 1 folder before, so all of the packaging props/targets ended up there. Now i created separate folders for WindowsAI/WindowsAI Interop

**Pull request merged**